.github/workflows/code-engine-preview.yml

```diff
@@ -0,0 +1,112 @@
+# This code is a Qiskit project.
+#
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+name: Docker image deployment process in IBM Cloud Code Engine
+
+on:
+  workflow_call:
+    inputs:
```

Comment:
I removed every input except for `env_name` because they were either unused or can safely be hardcoded.